assets/js/components/KeyMetrics/ConfirmSitePurposeChangeModal.js

```diff
@@ -135,35 +135,27 @@ function ConfirmSitePurposeChangeModal( {
 		).getUserInputPurposeConversionEvents();
 	} );
 
-	const {
-		saveUserInputSettings,
-		setKeyMetricsSetting,
-		saveKeyMetricsSettings,
-	} = useDispatch( CORE_USER );
+	const { setUserInputSetting, saveUserInputSettings } =
```

Comment:
Here and in user-input component setting is saved in user input settings within site scope instead of user specific key metrics setting